contrib/building.md

```diff
@@ -35,7 +37,7 @@
       * [Debugging failing Github Actions CI tests](#debugging-failing-github-actions-ci-tests)
    * [Testing against real apps](#testing-against-real-apps)
 
-<!-- Added by: tom.duncalf, at: Tue 23 Nov 2021 10:03:05 GMT -->
+<!-- Added by: tom.duncalf, at: Thu 25 Nov 2021 12:07:16 GMT -->
```

Comment:
Is this comment block necessary?  We have git blame for this 😆

Reply:
It's added by the `gh-md-toc` tool... but we can suppress it actually, I can do a separate PR for that updating the command too

Reply:
In fact I had to do a rebase anyway so I added it to this PR :) https://github.com/realm/realm-js/pull/4107/commits/2380a7fdb4685d6d664e2a2d1fe16ca48a414fab

Reply:
Ah, that explains it.  Cool tool